lib/rules/web/admin_console/4.4/search.xyaml

```diff
@@ -0,0 +1,44 @@
+search_by_resource:
+  elements:
+  - selector: 
+      xpath: //button[contains(., 'Resources')]
+    op: click
+  - selector:
+      xpath: //li//span[text()='<resource_kind>'] | //li//span[text()=<resource_kind>]/following-sibling::span[contains(., '<resource_group>')]
+    op: click
+    timeout: 15
+clear_search_item:
```

Comment:
`clear_resource_item_filter`

Reply:
or `clear_one_resource_filter`, another one is `clear_all_resources_filter`, then both of them looks much clear